setup.py

```diff
@@ -1,16 +1,13 @@
-
 from setuptools import find_packages, setup
 
-import openscap_report
-
 
 def get_long_description():
     with open("README.md", "r", encoding="utf-8") as readme_file:
         return readme_file.read()
 
 
-setup(name='openscap_report',
-      version=openscap_report.__version__,
+setup(name='openscap-report',
+      version='0.1.0',
```

Comment:
Why don´t you use the version from the variable (`openscap_report.__version__`)?

Reply:
If I remember correctly it was because of PyPI.